Sources/NIOPosix/SocketChannel.swift

```diff
@@ -440,10 +437,7 @@ final class DatagramChannel: BaseSocketChannel<Socket> {
     init(socket: Socket, parent: Channel? = nil, eventLoop: SelectableEventLoop) throws {
         self.vectorReadManager = nil
         try socket.setNonBlocking()
-        self.pendingWrites = PendingDatagramWritesManager(bufferPool: eventLoop.bufferPool,
-                                                          msgs: eventLoop.msgs,
-                                                          addresses: eventLoop.addresses,
-                                                          controlMessageStorage: eventLoop.controlMessageStorage)
+        self.pendingWrites = PendingDatagramWritesManager(bufferPool: eventLoop.bufferPool, msgBufferPool: eventLoop.msgBufferPool)
```

Comment:
Can we save the `msgBufferPool` on `self` here? That way we can avoid reaching into the EL to grab it later on, which is a nice win.

Reply:
That change was needed to refer controlMessageStorage from the PooledMsgBuffer. Now controlMessageStorage moved back to the EL, so probably that change not needed any more. What do you think?